<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QThesisMainWindowClass</class>
 <widget class="QMainWindow" name="QThesisMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dublin Traffic Light Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>462</width>
      <height>443</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>102</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Intersection: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="combo_intersection">
         <property name="minimumSize">
          <size>
           <width>349</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Time Span</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Start Time:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTimeEdit" name="time_start"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>End Time: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTimeEdit" name="time_end">
            <property name="locale">
             <locale language="English" country="Ireland"/>
            </property>
            <property name="maximumTime">
             <time>
              <hour>23</hour>
              <minute>45</minute>
              <second>0</second>
             </time>
            </property>
            <property name="currentSection">
             <enum>QDateTimeEdit::HourSection</enum>
            </property>
            <property name="calendarPopup">
             <bool>false</bool>
            </property>
            <property name="time">
             <time>
              <hour>23</hour>
              <minute>45</minute>
              <second>0</second>
             </time>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Number of Test Runs</string>
         </property>
         <widget class="QSpinBox" name="simulation_runs">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="group_specialdays">
         <property name="title">
          <string>Special Days</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="days_all">
            <property name="text">
             <string>All</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="days_weekdays">
            <property name="text">
             <string>Weekdays</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="days_weekends">
            <property name="text">
             <string>Weekends</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="days_holidays">
            <property name="text">
             <string>Holidays</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="trafficlights_group">
         <property name="title">
          <string>Traffic Light Control</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="tls_fixed">
            <property name="text">
             <string>Fixed-time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="tls_actuated">
            <property name="text">
             <string>Actuated</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="tls_adaptive">
            <property name="text">
             <string>Adaptive</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="group_weather">
         <property name="title">
          <string>Weather Conditions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="weather_all">
            <property name="text">
             <string>All</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="weather_rain">
            <property name="text">
             <string>Rain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="weather_dry">
            <property name="text">
             <string>No Rain</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Date Span</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Start Date: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="date_start"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>End Date: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="date_end"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>348</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="start_button">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qthesismainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
